connect/runtime/src/test/java/org/apache/kafka/connect/runtime/isolation/PluginScannerTest.java

```diff
@@ -37,13 +36,9 @@
 
 public class PluginScannerTest {
 
-    private enum ScannerType { Reflection, ServiceLoader }
```

Comment:
None of the changes in this test are necessary for the PR. Just cleaning things up by removing unused code and an unnecessary newline while I was in the neighborhood debugging some test failures in this suite while iterating locally.